periodic/style.css

```diff
@@ -0,0 +1,126 @@
+body {
+	background-color: black;
+}
+
+.table {
+	width: 73em;
+	height: auto;
+	padding: 10px;
+	margin: auto;
+	padding-top: 10px;
+  }
+  
+  .elem {
+	display: inline-block;
+	position: relative;
+	width: 80px;
+	height:80px;
+	margin: 3px;
+	border: 2px solid white;
+	transition: 0.2s;
+  }
+
+  .legend-other {
+	display: inline-block;
+	position: relative;
+	width: 182px;
+	height: 50px;
+	margin: 3px;
+  }
+
+  .box {
+	display: inline-block;
+	position: relative;
+	width: 20px;
+	height: 20px;
+	margin: 3px;
+	border: 2px solid white;
+	transition: 0.3s;
+  }
+
+  .text-box {
+	display: inline-block;
+	position: absolute;
+	width: auto;
+	height: 25px;
+	margin: 5px;
+	padding-bottom: 1px;
+	font-family: sans-serif;
+	font-size: 90%;
+	color: white;
+  }
+
+  .elem:hover {
+	  box-shadow: 2px 2px 5px black;
+	  cursor: pointer;
+	  transform: scale(2);
+	  z-index: 1000;
+	  filter:saturate(250%)
```

Comment:
Never seen this filter, cool!

Reply:
Thanx! Yes, very much a fan of football! Favourite teams in Europe are: AEK Athens (Greece), did you see that AEK got 3 Ukrainian players?  nr 19. Chygrinskij, 28. Shakov & 33. Danchenko! Crystal Palace (in UK), GIF Sundsvall in Sweden & Glasgow Rangers in Scotland!